osu.Game.Rulesets.Taiko.Tests/Skinning/TestSceneDrawableTaikoMascot.cs

```diff
@@ -175,25 +165,27 @@ private void setBeatmap(bool kiai = false)
             scoreProcessor.ApplyBeatmap(Beatmap.Value.Beatmap);
         }
 
-        private void createDrawableRuleset()
+        private void prepareTest(bool kiai)
         {
-            AddUntilStep("wait for beatmap to be loaded", () => Beatmap.Value.Track.IsLoaded);
+            AddStep("set beatmap", () => setBeatmap(kiai));
 
             AddStep("create drawable ruleset", () =>
             {
-                Beatmap.Value.Track.Start();
-
                 SetContents(_ =>
                 {
                     var ruleset = new TaikoRuleset();
                     return new DrawableTaikoRuleset(ruleset, Beatmap.Value.GetPlayableBeatmap(ruleset.RulesetInfo));
                 });
             });
+
+            AddUntilStep("wait for track to be loaded", () => MusicController.TrackLoaded);
+            AddStep("start track", () => MusicController.CurrentTrack.Restart());
```

Comment:
This was an important addition as previously the track may not have been restarted between tests (causing time to eventually veer too far into the future).

I believe this only happens with my upcoming clock changes due to beatmap equality meaning the same beatmap is used between tests, but seems good to add for safety.